w3c_validator.py

```diff
@@ -103,5 +114,4 @@ def __analyse(file_path):
         __print_stderr("usage: w3c_validator.py file1 file2 ...\n")
         exit(1)
 
-    for file_path in sys.argv[1:]:
-        __analyse(file_path)
+    sys.exit(__main_app())
```

Comment:
Please at a comment before this line and in the doctring on top of the file for explaining what the exit code will be = number of errors